assets/styles/_base.scss

```
@@ -263,51 +309,88 @@ button {
   line-height: inherit;
   padding: 0;
   text-align: inherit;
+
+  &:focus {
+    outline-color: var(--theme-color-accent-alpha);
+  }
 }
 
 .button {
   appearance: none;
-  background-color: $color-accent;
-  border: 1px solid $color-accent;
+  background-color: var(--theme-color-accent);
+  border: 1px solid var(--theme-color-accent);
   border-radius: 4px;
-  color: $color-background;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.25);
+  color: var(--theme-color-background);
   cursor: pointer;
   display: inline-block;
   font-family: inherit;
-  font-size: 0.875rem;
+  font-size: 1rem;
   font-weight: 700;
   line-height: inherit;
-  padding: 1rem 1.5rem;
+  outline: none;
+  padding: 0.875rem 1.5rem;
   text-align: center;
   text-decoration: none;
-  text-transform: uppercase;
-  transition: background-color 0.2s ease, border-color 0.2s ease, color 0.2s ease;
+  transition: background-color 0.2s ease, border-color 0.2s ease, color 0.2s ease, box-shadow 0.2s ease;
```

Could we use a shortcut?

```suggestion
  transition: all 0.2s ease;
```